PillarChap/Pillar.pillar

```diff
@@ -598,10 +796,58 @@ To create a preformatted block, begin each line with ==\===. A preformatted bloc
 
 !!!Commented lines
 
-Lines that start with a ==%== are considered comments and will not be rendered in the resulting document.
+Lines that start with a ==%== are considered comments and will not be rendered in the resulting document, except during a Pillar export.
 
-!! Configuring your output
+!! Write a presentation
+@sec:presentationPillar
+
+Since the version 0.59 you have the possibility to write presentations with Pillar.
+For now you can export a Pillar file as a ==beamer== or a ==deck.js== file.
 
+With the default te;plate you can provide some meta-informations to generate the first slide.
+For example:
+
+[[[
+{
+  "title":"Booleans and Conditions",
+  "subtitle":"",
+  "author":"Stephane Ducasse and Damien Cassou",
+  "complement":"http://stephane.ducasse.free.fr/ stephane.ducasse@inria.fr"
+}
+]]] 
+
+In a presentation export the headers create the structure of your presentation. Each title create a slide with the text in it. 
```

Comment:
In a presentation, Pillar headers (using the ==! Title== syntax) generate the structure of the presentation (sections and subsections). Each definition of a section generates a slide with the section title.